Assets/Scripts/CoRoutine/CoRoutine.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CoRoutine : MonoBehaviour
```

Comment:
I think you need to rename this class to a more generic one because your class name is not actually justifying what you have written.